core/src/trezor/pin.py

```diff
@@ -10,6 +10,18 @@
 _started_with_empty_loader = False
 keepalive_callback: Any = None
 
+_ignore_loader_messages: tuple[str, ...] | None = None
```

Comment:
to simplify, make it always a tuple and set to `()` when there are no ignored messages

Reply:
Sure, reverted in [00263df](https://github.com/trezor/trezor-firmware/pull/2840/commits/00263dfc3b161bd142a44d9cb0b17935784e35ae)